### DISCUSSION TITLE
v2.14.4

### DISCUSSION THREAD

**Discussion by Mygod** [Mygod/VPNHotspot#391] [General]
Fixes.

**Full Changelog**: https://github.com/Mygod/VPNHotspot/compare/v2.14.3...v2.14.4

<hr /><em>This discussion was created from the release <a href='https://github.com/Mygod/VPNHotspot/releases/tag/v2.14.4'>v2.14.4</a>.</em>

**Comment by drnmx** [comment]
Thank you for this application, an application that I can not do without every day ❤️
